uaclient/clouds/identity.py

```diff
@@ -84,7 +100,7 @@ def cloud_instance_factory() -> clouds.AutoAttachCloudInstance:
         "gce": gcp.UAAutoAttachGCPInstance,
     }  # type: Dict[str, Type[clouds.AutoAttachCloudInstance]]
 
-    cloud_type = get_cloud_type()
+    cloud_type, _error = get_cloud_type()
```

Comment:
I think we can just ignore the variable, right ?
`cloud_type, _ = get_cloud_type()`